packages/transform/src/resources.ts

```diff
@@ -33,10 +33,10 @@ export class TransformControlsStore {
 
 @struct
 export class CanvasRect {
```

Comment:
The initialization here is probably what caused the most recent tab freezing you saw, by the way. Previous API wasn't tolerant of this, new one is 😄